src/current/v23.2/transactions.md

```diff
@@ -228,27 +228,14 @@ For more information about the relationship between these levels, see [A Critiqu
 
 You can limit the number of rows written or read in a transaction at the cluster or session level. This allows you configure CockroachDB to log or reject statements that could destabilize a cluster or violate application best practices.
 
-Use the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.defaults.transaction_rows_written_log`,
-`sql.defaults.transaction_rows_written_err`, `sql.defaults.transaction_rows_read_log`, and
-`sql.defaults.transaction_rows_read_err` and [session settings]({% link {{ page.version.version }}/set-vars.md %}) `transaction_rows_written_log`,
-`transaction_rows_written_err`, `transaction_rows_read_log`, and
-`transaction_rows_read_err` to limit the number of rows written or read in a
-transaction. When the `log` limit is reached, the transaction is logged to the `SQL_PERF` channel.
-When the `err` limit is reached, the transaction is rejected. The limits are enforced after each
-statement of a transaction has been fully executed.
-
-The "write" limits apply to `INSERT`, `INSERT INTO SELECT FROM`, `INSERT ON CONFLICT`, `UPSERT`, `UPDATE`,
-and `DELETE` SQL statements. The "read" limits apply to the `SELECT`
-statement in addition to the statements subject to the "write" limits. The limits **do not**
-apply to `CREATE TABLE AS`, `SELECT`, `IMPORT`, `TRUNCATE`, `DROP`, `ALTER TABLE`, `BACKUP`,
-`RESTORE`, or `CREATE STATISTICS` statements.
+{% include {{ page.version.version }}/sql/transactions-limit-rows.md %}
+
+The limits are enforced after each statement of a transaction has been fully executed. The "write" limits apply to `INSERT`, `INSERT INTO SELECT FROM`, `INSERT ON CONFLICT`, `UPSERT`, `UPDATE`, and `DELETE` SQL statements. The "read" limits apply to the `SELECT` statement in addition to the statements subject to the "write" limits. The limits **do not** apply to `CREATE TABLE AS`, `SELECT`, `IMPORT`, `TRUNCATE`, `DROP`, `ALTER TABLE`, `BACKUP`, `RESTORE`, or `CREATE STATISTICS` statements.
```

Comment:
`SELECT` is also listed in the section where the limits do not apply.

Reply:
Removed.